src/libraries/System.IO.FileSystem/tests/System.IO.FileSystem.Tests.csproj

```diff
@@ -5,6 +5,7 @@
     <TargetFrameworks>$(NetCoreAppCurrent)-windows;$(NetCoreAppCurrent)-Unix;$(NetCoreAppCurrent)-Browser</TargetFrameworks>
 
     <WasmXHarnessMonoArgs>--working-dir=/test-dir</WasmXHarnessMonoArgs>
+    <StringResourcesPath>../../System.Private.CoreLib/src/Resources/Strings.resx</StringResourcesPath>
```

Comment:
Why is it not needed for `System.IO.FileSystem.Net5Compat.Tests.csproj`?

Reply:
> Why is it not needed for `System.IO.FileSystem.Net5Compat.Tests.csproj`?

Good catch. I don't know why we weren't seeing a failure. Let us verify.

Reply:
We also don't need it in `System.IO.FileSystem.Tests.csproj`, it was first added because we placed a helper (`VerifyValidPath`) in `PathInternal` but we noticed that this would make that `PathInternal` takes a dependency in the resx file and instead we moved it somewhere else.